core/sets/sets-docs.factor

```diff
@@ -46,7 +46,7 @@ ARTICLE: "set-operations" "Operations on sets"
 "Operations on groups of sets:"
 { $subsections
     combine
```

Comment:
I still see ``combine`` is some of these diffs?

Reply:
...that is correct, because I forgot to amend the diff prior to pushing. Fixed.